prelude/http_archive/http_archive.bzl

```diff
@@ -47,7 +47,7 @@ def _unarchive_cmd(
         ext_type: str,
         exec_is_windows: bool,
         archive: Artifact,
-        strip_prefix: [str, None]) -> cmd_args:
+        strip_prefix: [str, None]) -> tuple.type:
```

Comment:
I think you can write this as `tuple[cmd_args, bool]`. When there is a complex return type it's often useful to document on the function what the return type means.

Reply:
I tried it as `tuple[cmd_args, bool]` and it didn't work: https://app.circleci.com/pipelines/github/facebook/buck2/11065/workflows/07b21140-63be-4cd5-82e3-4a6d88c98cfa/jobs/28597

From searching the rest of the code base it seems the only possibility is simply `tuple`.